exchange_server/assets/dashboards/overview.json

```
@@ -0,0 +1,805 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Exchange Server Overview",
+    "read_only": false,
+    "description": "## Exchange Server\n\nThis dashboard provides a high-level overview of your Exchange Server instances so you can monitor metrics related to network, storage and workers.\n\n[Official integration docs](https://docs.datadoghq.com/integrations/exchange_server/)\n\nClone this template to make changes and add your own graphs and widgets.\n",
```

```suggestion
    "description": "## Exchange Server\n\nThis dashboard provides a high-level overview of your Exchange Server instances so you can monitor metrics related to network, storage, and workers.\n\n[Official integration docs](https://docs.datadoghq.com/integrations/exchange_server/)\n\nClone this template to make changes and add your own graphs and widgets.\n",
```
Small change then 👍 for docs